### ISSUE TITLE
Publish only after manual check

### ISSUE THREAD

**Issue by Matthias84** [mapillary/mapillary_issues#1518]
I consider privacy as mission critical (in germany we had a lot of discussions about Google streetview) to the acceptance on a photo VGI project. Currently a lot of plates and faces aren't blured, which will IMHO cause bad reputation and legal problems.

So I recommend to enforce an manual check by the user for every image, BEFORE it's getting public to the website.

**Comment by jfburke3** [comment]
Maybe that should be a country-dependent feature.  There are some weekends when I shoot 10,000-15,000 images.  I'll never have time to go through them all, and since privacy laws for things in "plain view" in public aren't as strict in the U.S., I don't know that such a feature is necessary here.

Of course, there might be problems when photos are taken very close to a border to a country with less (or more) strict laws, as the infamous "boundary problem" might become an issue....

**Comment by Matthias84** [comment]
Of course it's somewhat a personal perspective. On the other hand 'with great power comes great responsibility', so the user is (in first place) responsible what he records and publishes.

**Comment by jesolem** [comment]
Good points @Matthias84 

For large sets this will be very hard and a lot of work pre-upload but I think there are some solutions we can consider. 
- Encourage people to quick review their sequences after they have been processed. 
- Blurring in the apps before uploading (tap to blur in the photo viewer) which would blur the original destructively
- Setting on your profile page to not publish your uploaded sequences before you review and OK them on web.

Besides this we are continuously improving the blur algos and @ybkuang is pushing out a major improvement soon.

**Comment by rudolfmayer** [comment]
I agree that we should specifically protect the privacy of those seen in the pictures.

I normally sort out pictures where you really mostly see just people, but not actually the road/landscape, before uploading. I do that locally on my PC, where I can scan through pictures very quickly and still recognise the contents on the big screen available; however, I think it is rather impractical for larger volumes of pictures to do that on the phone (screen too small, speed to slow) or after uploading (speed of the website definitely too slow...)

So maybe indeed if something like this is introduced, do it country-specific; but in first place, rather try to tune the blurring algorithm, and potentially make that one too aggressive to also blur false positives, which an anyhow be unblurred manually (later & on demand(

**Comment by peterneubauer** [comment]
We have just been the latest android apps disabled auto upload, letting the user mark sequences manually for upload, possibly after reviewing , and enabling auto upload manually. We hope that moves along these lines also.

**Comment by Matthias84** [comment]
Sounds good, thanks @peterneubauer

**Comment by richlv** [comment]
could this be a duplicate of #436 ?
